### ISSUE TITLE
Error code 403

### ISSUE THREAD

**Issue by AureliuszBrussy** [allegro/allegro-api#9258]
### Discussed in https://github.com/allegro/allegro-api/discussions/9256

<div type='discussions-op-text'>

<sup>Originally posted by **Sinful957** June 20, 2024</sup>
Hello, I have registered an API app (app 787) for my seller account connection, but I'm not able to connect and getting an error 403: Error code 403 {'errors': [{'code': 'OfferAccessDeniedException', 'message': 'Forbidden', 'details': None, 'path': '', 'userMessage': 'Access denied. You are not authorized to perform operations on the indicated offer', 'metadata': {} }]} App developers advised me to contact with the support and ask to activate API, as sometimes it's not actually activated. Could you check my API please?</div>

**Comment by AureliuszBrussy** [comment]
@Sinful957 typically, for this error two cases are triggered:

1. We return such a message because the user account that tries to create an offer is a regular account. Only company accounts can create offers using the API.
2. You are trying to edit an offer that belongs to someone else - it was created from a different account. 

If you still do not know what is wrong - please send the trace-id of the request. You can find it in the header of our response.

**Comment by Sinful957** [comment]
> @Sinful957 typically, for this error two cases are triggered:
> 
> 1. We return such a message because the user account that tries to create an offer is a regular account. Only company accounts can create offers using the API.
> 2. You are trying to edit an offer that belongs to someone else - it was created from a different account.
> 
> If you still do not know what is wrong - please send the trace-id of the request. You can find it in the header of our response.

Hello,

1. Look like I'm on a regular account. I'm a self employed and I couldn't register a business account. I spoke to Allegro representative about the limitations and I was told that there wouldn't be any difference between the accounts, it occurs.
2. That was a new product.

Is there any option to request the ability to create new products for a regular account through the API? I previously spoke with app developers and they say that this solution would be an option if allegro shows a goodwill.

**Comment by AureliuszBrussy** [comment]
No, it is not possible to create an offer using the API without a company account, we are not able to make an exception. 

If you want to test the API, you can freely use our [Sandbox](https://allegro.pl.allegrosandbox.pl/). You can [register a company account](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/nowe-konto?origin_url=%2F) using fake data. This will enable you to create offers on Sandbox through the API. 

Resource address - https://api.allegro.pl.allegrosandbox.pl/
Application registration - https://apps.developer.allegro.pl.allegrosandbox.pl/
Authorization address - https://allegro.pl.allegrosandbox.pl/auth/oauth/
